packages/shared/src/components/post/SquadPostContent.tsx

```diff
@@ -23,6 +23,7 @@ import { SourcePermissions, Squad } from '../../graphql/sources';
 import EditIcon from '../icons/Edit';
 
 const ContentMap = {
+  [PostType.Freeform]: WelcomePostContent,
```

Comment:
What does it mean? What is `WelcomePostContent`?

Reply:
This is just a placeholder to test it, but basically, we have a dedicated ticket to introduce a `PostContent` dedicated to FreeformPost.